src/Http/Routing/ref/Microsoft.AspNetCore.Routing.Manual.cs

```diff
@@ -367,6 +367,7 @@ internal partial class DefaultLinkParser : Microsoft.AspNetCore.Routing.LinkPars
         [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
         internal readonly partial struct MatcherState
         {
+            private readonly object _dummy;
```

Comment:
nit: This added field just doesn't matter. I missed the `Matcher` field when doing stuff in the 3.1 merge PR. Leave this alone as long as the validation succeeds.